queries/insertMovie.js

```diff
@@ -0,0 +1,16 @@
+import pool from "../database/db.js";
+
+const insertMovie = async ({ title, summary, imdbLink, rating }) => {
+  if (!title) {
+    throw new Error("Title is required");
+  }
+
+  const insertQuery =
+    "insert into movies (Title, Summary, IMDBLink, Rating) values ($1, $2, $3, $4) returning *;";
+  const values = [title, summary, imdbLink, rating];
+
+  const res = await pool.query(insertQuery, values);
+  return res.rows;
+};
```

Comment:
I would also add some validation for `imdbLink` and `rating`. For instance, I would make sure the link was actually a link from imdb, and for the rating, I would make sure it's a number, and between 1 & 5. I know you already added that validation on the database, but you also want to check on the server because it's a quicker check than having the database check the validation